features/roster.feature

```diff
@@ -0,0 +1,67 @@
+Feature: Roster
+
+  As a logged in users,
+  So that I can display my roster,
+  I can add new participant to my roster.
+
+  Background: user is logged in
+    Given a user with the email "aaronli98@berkeley.edu" with password "password" and with username "aaronli98" exists
+    And a registered user with the username "aaronli98" has a project named "Test Meeting 1" with id "1"
+    And a registered user with the username "aaronli98" has a project named "Test Meeting 2" with id "2"
+    And I am on the login page
+    When I fill in "Email" with "aaronli98@berkeley.edu"
+    And I fill in "Password" with "password"
+    And I press "Log In"
+    Then I should be on the projects page
+    And I should see "aaronli98"
+
+  Scenario: successfully create a new project
```

Comment:
We don't need these tests because they don't test roster-specific things, and I think they were covered in other tests. If not then ideally they should be moved over but this is fine. 
Same for the "user is logged in" scenario above, but again it's fine.

Reply:
Isn't the background necessary for testing the roster, though? I had to log in and stuff as a background thing in my tests for the matchings controller because it wouldn't be visible otherwise.

Reply:
Oh you're right, the background is totally necessary, I just read too quickly.

But I think the "Scenario: successfully create a new project" is still not appropriate here.